objmodel/chapter.md

```diff
@@ -885,44 +878,44 @@ class Instance(Base):
             self.map = new_map
 ````
 
-The class now passes ``None`` as the fields dict to ``Base``, as ``Instance``
-will now store the content of the dict in another way. Therefore it now needs
-to override the the ``_read_dict`` and ``_write_dict`` methods. In a real
-implementation, we would refactor the ``Base`` class to not be responsible for
-storing the field dictionary any more, but for this section having instances
+The class now passes ``None`` as the fields dictionary to ``Base``, as ``Instance``
+will store the content of the dictionary in another way. Therefore it needs
```

Comment:
I think in most cases here we should expand to "dictionary", when we're referring to the fields dictionary data structure. I agree with Mike that if we're talking about Python's `dict` the abbreviation is appropriate, but most of the time here we're referring to a dictionary implemented using `dict`.